DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/hardware_reuse/README.md

```diff
@@ -0,0 +1,358 @@
+# `Hardware Reuse` Sample
+
+This sample is a tutorial that demonstrates how to reuse hardware in your FPGA designs by using loops and task sequences.
+
+| Area                 | Description
+|:--                   |:--
+| What you will learn  | How to code your FPGA SYCL program to reuse hardware that has same functionality
+| Time to complete     | 30 minutes
+| Category             | Concepts and Functionality
+
+## Purpose
+
+In normal operation, the oneAPI FPGA compiler automatically in-lines repeated function calls, creating multiple instances of hardware that can run in parallel. This in-lining strategy is necessary to achieve high throughput, but in cases where you want to minimize area utilization, you may wish to sacrifice throughput and allow certain functions to be re-used. This sample demonstrates two strategies to re-use hardware in your designs:
+
+1. Calling a function in a loop will cause it to only be instantiated once
+2. Launching a task sequence with a function callback lets you re-use a function in cases where it is not practical to re-use it in a loop.
+
+## Prerequisites
+
+| Optimized for        | Description
+|:---                  |:---
+| OS                   | Ubuntu* 20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10 <br> Windows Server* 2019
+| Hardware             | Intel® Agilex® 7, Arria® 10, and Stratix® 10 FPGAs
```

Comment:
Intel® Agilex® -> Agilex™

Reply:
remain Intel branding here until whole code sample to be refreshed in once